nameko/testing/websocket.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import absolute_import
+
+import uuid
+import json
+
+from collections import defaultdict
+
+from eventlet.event import Event
+from eventlet.queue import Queue
+
+from nameko.exceptions import deserialize
+
+
+def make_virtual_socket(host, port, path='/ws'):
+    from websocket import WebSocketApp
```

Comment:
``websocket`` is in requirements.txt now and the docs talk about websockets as "built-in"